homeassistant/components/climate/strings.json

```diff
@@ -270,6 +270,9 @@
     "temp_out_of_range": {
       "message": "Provided temperature {check_temp} is not valid. Accepted range is {min_temp} to {max_temp}."
     },
+    "low_temp_higher_than_high_temp": {
+      "message": "Target low temperature can not be higher than target high temperature."
```

Comment:
The parameters are named `"Target temperature low"` and `"Target temperature high"` in the translations strings for the service.